src/screens/Home.js

```diff
@@ -4,7 +4,7 @@ import axios from 'axios';
 import Snackbar from '@material-ui/core/Snackbar';
 import Alert from '@material-ui/lab/Alert';
 import Grid from '@material-ui/core/Grid';
-import { Container } from '@material-ui/core';
+import { CircularProgress, Container } from '@material-ui/core';
```

Comment:
```suggestion
import { Container } from '@material-ui/core';
```

Reply:
Cannot remove CircularProgress as it is the component that has been called